go/tosca/revisions.go

```diff
@@ -36,6 +36,17 @@ func (r Revision) String() string {
 	}
 }
 
+func GetAllKnownRevisions() []Revision {
```

Comment:
nit: would a loop based on an upper limit constant be more future-proof?

Reply:
Done